prymer/api/picking.py

```diff
@@ -143,7 +143,8 @@ def build_primer_pairs(
             for rp in right_primers:
                 amp_span = PrimerPair.calculate_amplicon_span(lp, rp)
 
-                if amp_span.length > amplicon_sizes.max:
+                # Ignore pairings with amplicon sizes out of the range specified
+                if not amplicon_sizes.min <= amp_span.length <= amplicon_sizes.max:
```

Comment:
**note** In the future we could add a `.contains(x: T) -> bool` method or similar to `MinMaxOpt[T]`, which returns `True` if `x` is within the bounds of the min and max.

Reply:
This is a nice idea.  I'll log an issue.